README.md

```diff
@@ -69,9 +69,9 @@ and run it under `$USER`, in this case you will not be able to login as someone
 ### Deploy Teleport
 If you wish to deploy Teleport inside a Docker container:
 ```
-# This command will pull the Teleport container image for version 5.0
+# This command will pull the Teleport container image for version 6.0
 # Replace 5.0 with the version you need:
-$ docker pull quay.io/gravitational/teleport:5.0
+$ docker pull quay.io/gravitational/teleport:6.0
```

Comment:
There's no `6.0` image, might be worth changing this to `6`

https://quay.io/repository/gravitational/teleport?tag=latest&tab=tags

Reply:
Good catch, I've updated it.